acj/api/login.py

```diff
@@ -25,9 +25,21 @@ def login():
     else:
         permissions = authenticate(user)
 
-        if sess.get('LTI') and sess.get('oauth_create_user_link'):
+        if sess.get('LTI') and sess.get('oauth_create_user_link') and sess.get('lti_user') and sess.get('lti_context'):
```

Comment:
These should be two different if conditions (sorry I forgot to mention this last time)
`if sess.get('LTI') and sess.get('oauth_create_user_link')` links the user to lti_user if they aren't linked already

the other if statement can be interested before line 31 with `if sess.get('LTI') and sess.get('lti_context'):` which enrolls the user into the course